terraform/hetzner_cloud/user_data.yml

```diff
@@ -0,0 +1,29 @@
+#cloud-config
+runcmd:
+  # replace with the latest version
+  - wget -c https://github.com/Arriven/db1000n/releases/download/v0.8.2/db1000n-v0.8.2-linux-amd64.tar.gz -O - | tar -xz
```

Comment:
I didn't find a proper way for cloud init to pass a variable with version, so it's hardcoded in YAML, not a big deal